--- ViewServiceProvider.php	2022-02-18 21:48:20.125405427 +0100
+++ ViewServiceProvider.php	2022-02-18 21:47:53.720961792 +0100
@@ -52,10 +52,6 @@

             $factory->share('app', $app);

-            $app->terminating(static function () {
-                Component::forgetFactory();
-            });
-
             return $factory;
         });
     }
@@ -92,9 +92,6 @@
             return tap(new BladeCompiler(
                 $app['files'],
                 $app['config']['view.compiled'],
-                $app['config']->get('view.relative_hash', false) ? $app->basePath() : '',
-                $app['config']->get('view.cache', true),
-                $app['config']->get('view.compiled_extension', 'php'),
             ), function ($blade) {
                 $blade->component('dynamic-component', DynamicComponent::class);
             });
@@ -162,10 +156,6 @@
         $resolver->register('blade', function () {
             $compiler = new CompilerEngine($this->app['blade.compiler'], $this->app['files']);

-            $this->app->terminating(static function () use ($compiler) {
-                $compiler->forgetCompiledOrNotExpired();
-            });
-
             return $compiler;
         });
     }

